html.go

```diff
@@ -70,6 +74,9 @@ type HtmlRendererParameters struct {
 	HeaderIDPrefix string
 	// If set, add this text to the back of each Header ID, to ensure uniqueness.
 	HeaderIDSuffix string
+	// SanitizedAnchorNameOverride is an optional func to override
+	// the behavior for creating sanitized anchor names.
+	SanitizedAnchorNameOverride SanitizedAnchorNameFunc
```

Comment:
This PR should be about adding the _option_ to change the default. These 3 lines are good.